nss-day-cohort-30/bangazon-api-ambitious-albatrosses
# Added Employee and Department models

Fixes:
- Added Employee and Department models

Changes proposed in this request & reasons behind organizational or architectural decisions:
- We needed them

Steps to test:
- Compare contents of models to SQL database and ensure they match

System configuration (3rd party libraries to be installed, command line utilities to run, UI instructions, expected outcome):
- N/A

@ambitions-albatrosses
